layer/clients/data_catalog.py

```diff
@@ -185,22 +184,19 @@ def store_dataset(
 
     def initiate_build(
         self,
-        dataset: DatasetFunctionDefinition,
-        project_id: uuid.UUID,
-        is_local: bool,
+        project_id: ProjectId,
+        asset_name: str,
+        fabric: str,
```

Comment:
We have `layer/contracts/fabrics.py:class Fabric(enum.Enum):`, but we use `str` in many places..

Reply:
Yes, agreed that's a good one to use more consistently.